db/migrate/20201006213104_create_movies.rb

```diff
@@ -0,0 +1,8 @@
+class CreateMovies < ActiveRecord::Migration[5.2]
+  def change
+    create_table :movies do |t|
+
```

Comment:
For now we created a Movies table w/ just ID. Once we have consumed the API, we will also add the API's movie ID.